xcolor.dtx

```diff
@@ -4066,25 +4054,42 @@
 \@tfor\@@tmp:={dvipdf}{dvipdfmx}{dvips}{dvipsone}{dviwin}%
               {emtex}{luatex}{pctex32}{pctexhp}{pctexps}{pctexwin}{pdftex}%
               {tcidvi}{textures}{truetex}{vtex}{xetex}{dvisvgm}\do
- {\edef\@@tmp{\noexpand\DeclareOption{\@@tmp}%
-    {\def\noexpand\Gin@driver{\@@tmp.def}}}%
+ {\edef\@@tmp{%
+   \noexpand\DeclareKeys
+     {
+      \@@tmp\space .code = {\def\noexpand\Gin@driver{\@@tmp.def}},
+      \@@tmp\space .usage=load
+     }}%
   \@@tmp}
 %    \end{macrocode}
 %
 % Drivers that are derivatives of others:
 %    \begin{macrocode}
-\DeclareOption{dviwindo}{\ExecuteOptions{dvipsone}}
-\DeclareOption{oztex}{\ExecuteOptions{dvips}}
-\DeclareOption{xdvi}{\ExecuteOptions{dvips,monochrome}}
-\DeclareOption{dvipdfm}{\ExecuteOptions{dvipdfmx}}
+\DeclareKeys
+ {
+  ,dviwindo .code   = {\SetKeys{dvipsone}}
+  ,dviwindo .usage  = load
+  ,oztex    .code   = {\SetKeys{dvips}}
+  ,oztex    .usage  = load
+  ,xdvi     .code   = {\SetKeys{dvips,monochrome}}
+  ,xdvi     .usage  = load
+  ,dvipdfm  .code   = {\SetKeys{dvipdfmx}}
+  ,dvipdfm  .usage  = load
+ }
 %    \end{macrocode}
 %
 % Options used by (some) drivers.
 %    \begin{macrocode}
-\DeclareOption{xcdraw}{\def\XC@@xcd@{1}}
-\DeclareOption{noxcdraw}{\def\XC@@xcd@{-1}}
-\DeclareOption{fixinclude}{\def\XC@@xcf@{1}}
-\DeclareOption{prologue}{\def\XC@@xcp@{1}}
+\DeclareKeys
+ {
+   ,xcdraw   .code    = {\def\XC@@xcd@{1}}
+   ,xcdraw   .usage   = load %TODO check
+   ,noxcdraw .code    = {\def\XC@@xcd@{-1}}
+   ,noxcdraw .usage   = load %TODO check
+   ,fixinclude .code  = {\def\XC@@xcf@{1}}
+   ,fixinclude .usage = load %TODO check
+   ,prologue .code    = {\def\XC@@xcp@{1}}
+   ,prologue .usage   = load %TODO check
 %    \end{macrocode}
```

Comment:
Do we really need to do anything for any of these today?

Reply:
I don't know (thats why there is a todo everywhere), but I think that should be discussed with the next version and not now.